gdcdictionary/schemas/somatic_aggregation_workflow.yaml

```diff
@@ -24,12 +24,21 @@ systemProperties:
   - state
 
 links:
-  - name: annotated_somatic_mutations
-    backref: somatic_aggregation_workflows
-    label: performed_on
-    target_type: annotated_somatic_mutation
-    multiplicity: one_to_many
+  - exclusive: false
     required: true
+    subgroup:
+      - name: annotated_somatic_mutations
+        backref: somatic_aggregation_workflows
+        label: performed_on
+        target_type: annotated_somatic_mutation
+        multiplicity: one_to_many
+        required: true
```

Comment:
This should be "false" I think, since we want either to work.

Reply:
updated